config/local_variables.tfvars

```diff
@@ -0,0 +1,3 @@
+base_url    = "http://localhost:3000"
+client_id   = "terraform-client"
```

Comment:
Is the client_id different to the one we plan to use in production? If not shouldn't this be stored within the secrets too.

Reply:
This has been removed, it may be worth removing base url though as it could expose the not-prod url